web-speech-api/audio-track-recognition/index.html

```diff
@@ -0,0 +1,27 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width" />
+
+    <title>Audio track recognition</title>
+
+    <link rel="stylesheet" href="style.css" />
+    <script defer src="script.js"></script>
+  </head>
+
+  <body>
+    <h1>Audio track recognition</h1>
+
+    <button>Recognize audio!</button>
```

Comment:
or 
```suggestion
    <button>Play a sample audio</button>
```
or maybe even "Play the audio and start speech recognition" but probably too long for button text. 
what about "Play audio and run recognition"?